modules/ProjectTask/ProjectTask.php

```diff
@@ -148,7 +148,7 @@ public function save($check_notify = false)
     {
         //Bug 46012.  When saving new Project Tasks instance in a workflow, make sure we set a project_task_id value
```

Comment:
Maybe this comment can be removed now, it's not really necessary and no longer makes sense.